linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_log_format/rule.yml

```diff
@@ -19,6 +19,7 @@ identifiers:
 
 references:
     ospp: FAU_GEN.1
+    srg: SRG-OS-000255-GPOS-00096
 
 ocil_clause: log_format isn't set to ENRICHED
```

Comment:
``ENRICHED`` is not a valid value per the man page (ref: https://linux.die.net/man/5/auditd.conf)

Reply:
@shawndwells this is outside the scope of this PR. Also I am not sure how up-to-date https://linux.die.net is because `man auditd.conf` shows ENRICHED as a valid optin.

Reply:
Enriched has been part of auditd since 2.5. It is a mandatory requirement for any remote logging because it resolves local items like user name, group name, which could be different on another machine. These get appended to each record so that interpretation is easier.